xarray/tests/test_plot.py

```diff
@@ -91,6 +90,7 @@ def contourf_called(self, plotmethod):
 
 class TestPlot(PlotTestCase):
 
+    @requires_matplotlib
```

Comment:
We don't need to move the `requires_matplotlib` decorator, since we don't rely on inheritance for enabling/disabling tests.